optaplanner-quickstart/src/main/java/org/acme/optaplanner/domain/Lesson.java

```diff
@@ -20,7 +19,7 @@
 
     @PlanningId
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue()
```

Comment:
I think we can remove the extra `()` for clarity

Reply:
I absolutely agree, let me update the PR next morning

Reply:
@gastaldi extra `()` are now removed